src/target/move.c

```diff
@@ -341,10 +344,22 @@ void move_front_many(int cells)
 	entered_next_cell();
 }
 
+/**
+ * @brief Move front a defined distance.
+ *
+ * @param[in] distance Distance to travel.
+ */
+void parametric_move_front(float distance, float end_linear_speed)
+{
+	enable_walls_control();
+	target_straight(get_encoder_average_micrometers(), distance,
+			end_linear_speed);
+}
+
 /**
  * @brief Move left or right into the next cell.
```

Comment:
include @param[in]

Reply:
4e0d4ddcb7824b6d9847b7600fc6053c53f2015f